src/extension.ts

```diff
@@ -232,14 +230,13 @@ export function activate(context: vscode.ExtensionContext) {
             )
 
             return action
-          } catch (error) {
+          } catch (error: any) {
```

Comment:
`unknown` and `as Error`?

Reply:
I’d rather not. These types were wrong and this PR changes nothing about that. It’s going to be fully replaced soon. I’m purely trying to make the diffs manageable by doing some chores that can be done before replacing the entire TypeScript code with the language server implementation.